CMakeLists.txt

```diff
@@ -21,16 +21,10 @@
 PROJECT(mudlet)
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
Because 3.1 knows about C++11.

Shouldn't be a problem for someone who wants to have people always use the latest version of Mudlet for their maps ;)